make/ReleaseFile.gmk

```diff
@@ -50,6 +50,7 @@ define create-info-file
   $(call info-file-item, "IMPLEMENTOR", "$(COMPANY_NAME)")
   $(call info-file-item, "OS_NAME", "$(RELEASE_FILE_OS_NAME)")
   $(call info-file-item, "OS_ARCH", "$(RELEASE_FILE_OS_ARCH)")
+  $(call info-file-item, "JAVA_VERSION", "$(VERSION_MAJOR)")
```

Comment:
This change isn't specific to OpenJ9. OpenJDK builds already contain "JAVA_VERSION", we should determine why OpenJ9 builds do not and fix this problem. I found jdk.tools.jlink.internal.plugins.ReleaseInfoPlugin which seems to add the "JAVA_VERSION" when running jlink. Not sure why OpenJ9 doesn't get the same.

Reply:
The build creates an intermediate JVM at a location like `openjdk/build/linux-x86_64-normal-server-release/jdk` with `make/ReleaseFile.gmk`, use this intermediate JVM to create other `modules`, and eventually invokes `jlink` to add `JAVA_VERSION` in final JDK location `openjdk/build/linux-x86_64-normal-server-release/images`.
This pull request allows the intermediate JVM to determine Java version with `release` file for `OpenJ9 Java 9` build, otherwise, `J2SE_LATEST | J2SE_SHAPE_LATEST` will be used which is for `Java 18.3` and can't work with `Java 9`.

Reply:
I'm very reluctant to accept a change to the extensions which isn't specific to OpenJ9 and would affect a plain OpenJDK build as well. All changes should be contributed to OpenJDK to avoid the need for any extensions, and we need to justify these changes and get them accepted. Can we figure out a different way to fix this problem?

Reply:
Updated the changeset to make the change specific to OpenJ9.